apps/angular/projection/src/app/component/teacher-card/teacher-card.component.ts

```diff
@@ -1,35 +1,60 @@
+import { AsyncPipe } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
-import { FakeHttpService } from '../../data-access/fake-http.service';
+import {
+  FakeHttpService,
+  randTeacher,
+} from '../../data-access/fake-http.service';
 import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
-import { Teacher } from '../../model/teacher.model';
+import { ListItemTemplateDirective } from '../../directive/list-item-template.directive';
 import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-teacher-card',
-  template: `<app-card
-    [list]="teachers"
-    [type]="cardType"
-    customClass="bg-light-red"></app-card>`,
+  template: `
+    <app-card
+      [list]="(store.teachers$ | async) ?? []"
+      class="bg-light-red"
+      (addItem)="addItem()">
+      <img src="assets/img/teacher.png" width="200px" />
+      <ng-template let-item list-item-template>
+        <app-list-item
+          [name]="item.firstname"
+          (deleteItem)="deleteItem(item.id)">
+        </app-list-item>
+      </ng-template>
+    </app-card>
+  `,
   styles: [
     `
-      ::ng-deep .bg-light-red {
+      .bg-light-red {
         background-color: rgba(250, 0, 0, 0.1);
       }
     `,
   ],
   standalone: true,
-  imports: [CardComponent],
+  imports: [
+    CardComponent,
+    ListItemComponent,
+    ListItemTemplateDirective,
+    AsyncPipe,
```

Comment:
you can use the new signal API to get read of this. Signal will be the new way of binding properties to the template, it's better to get use to it now. And many benefits will come in a very near future

Reply:
Yep, still in developer preview (as of v16) but could make use of it. Hoping that interop with RxJS is working as intented.